### DISCUSSION TITLE
Calculator Ideas for Section G? (New Buttons)

### DISCUSSION THREAD

**Discussion by CarlosPietrasanta** [CarlosPietrasanta/Tafe230130#11] [Q&A]
Hi guys,

I wonder what kind of button we can implement to do Section G, since being 3 in the group, we already implemented the Trip Calculator button in the original program.

From my understanding, each one of you should fork the repository, add a new button and raise a Pull Request to meet the grading criteria.

I was thinking something like BMI Calculator and Tip Calculator (Don't worry about the code! It's simply adding the button in the Main Menu). But let me know if you have any other ideas to implement instead of those two.

Thanks,
Carlos.

**Comment by ozdotcom** [comment]
Hi Carlos, Jason,
I have updated the main menu with the BMI calculator button.
Click event - message - BMI calculate C# code will be developed later.

Check it out and see what you think.
There is not much room for Jason to add another button under the BMI, Happy to move things around a bit if needed.
Let me know if I need to change anything.
Sharryn
ozdotcom

**Comment by Jasonlovedataanalyst** [comment]
I have  update the main menu with a MMG button , as well as add the message in the click event , could you help to review at your convenience 

![image](https://user-images.githubusercontent.com/87595451/228381918-e36c4e9c-63f9-46c3-a3a2-20e557ffd633.png)
![image](https://user-images.githubusercontent.com/87595451/228381929-665f9ac4-c3b0-4297-8226-e40206dffb2b.png)

**Reply by Jasonlovedataanalyst** [reply]
![image](https://user-images.githubusercontent.com/87595451/228382804-54b42561-e47b-486a-aae7-06f44a0321a1.png)
![image](https://user-images.githubusercontent.com/87595451/228382812-a9119cd2-eb35-404e-a8cd-c0b557bae14a.png)
